docker/puppetdb/docker-entrypoint.d/30-consul.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+if [ "$CONSUL_ENABLED" = "true" ]; then
+  ipaddress="$(ifconfig eth0 | grep 'inet addr' | cut -d ':' -f 2 | cut -d ' ' -f 1)"
```

Comment:
I was wondering about the availability of `ifconfig`, and noticed that it does not exist in the `ubuntu` and `centos` images. It does exist in `alpine` though.

So something else that we install in these images causes this to be brought in?

Just wanted to point it out, not sure there's anything to do about it though.

Reply:
Looks like you get `ifconfig` via the `net-tools` package on Ubuntu.